### ISSUE TITLE
Alumni Page

### ISSUE THREAD

**Issue by mzogana** [katisoseeiso/lems-website#1]
Thanks for the page @katisoseeiso . Just a few pointers/suggestions: 

- Please embed it to the Alumni tab so that on click, one gets redirected to the page. Simply replace the section scroller on the NavBar - <li><a href="#index.html" class="hvr-rectangle-out scroll">Alumni</a></li> to <li><a href="alumni.html" class="hvr-rectangle-out scroll">Alumni</a></li> 

- for typing simplicity, please also rename "Alumini.html" to "alumni.html".  

- The Banner area looks good (though we may need to source out a different image). 
- Please add move the button at the bottom section to beneath the "committed to excellence" text on the banner and have the button label read "Learn more" - this button will then link to the LEMSAN WEBSITE. 
- Turn that bottom horizontal strip into a mini-footer with LEMSAN contact details (whichever details you see fit) (we can adopt the Main footer and just swap LEMSAN details)

- The Vision, Mission...section can be replaced by a brief overview of LEMSA - It can be just a paragraph or two of what LEMSAN is about and how it works. 

- The Exec section: I'm think the LEMS Logos can be turned into round avatars with our head shots - it's only the four of us so we can source out our pics right away and have them up. Then the button contents can be Name, Surname (top line), second line (Portfolio/title), third line (Location) 

That's all from my side on the page, let me know your thoughts and/or if you have any questions. 

Thanks!

**Comment by katisoseeiso** [comment]
Comment 4 & 5

I kinda wanted people to go through the whole page and then go to the main website when they get to the bottom of the current page. And there they can find our details. (trying to move traffic to our site for people interested and not give them all essential information on this one)

**Comment by mzogana** [comment]
Yeah I'm with you on that.  So we can forget the learn more button mentioned above on the banner then. 

however, it's still important to have a brief overview nje of LEMSAN. The footer can then be of the main LEMS website across ALL pages. 

One thing I also forgot to mention, the nav bar has to be accross ALL pages too so that one can click any tab irrespective of which page they're on, so you may include it on this page too. 

Cheers,

**Comment by katisoseeiso** [comment]
Alright, I will add the LEMSAN brief. I have already added the nav-bar and embedded a google maps.

**Comment by mzogana** [comment]
Awesome! nice one with the map! 

Please keep your gMap API safe, we may need it further down the line, so since you already created one, we might as well just keep iterating it.  

When you have done the brief overview, feel free to "close" this issue. 

Thanks,
